htmlmaker_preprocessing.ps1

```diff
@@ -32,35 +35,45 @@ Function LogWrite
    Add-content $Logfile -value "$logstring"
 }
 
-# if original file name is a .doc file
-$fileType = ".doc"
-If ($filenameSplit -eq $filename + $fileType)
-{
-	# Converts a word .doc in bookmaker_tmp to a .docx
-	# so you have to have already run tmparchive.rb
-	write-host "Converting $filenameSplit to .docx from $fileType"
 
-	$SaveFormat = "microsoft.office.interop.word.WdSaveFormat" -as [type]
-	$word = New-Object -ComObject word.application
-	$word.visible = $false
+# Converts a word .doc in bookmaker_tmp to a .docx
+# so you have to have already run tmparchive.rb
+write-host "Converting $filenameSplit to .docx from $fileType"
+
+$SaveFormat = "microsoft.office.interop.word.WdSaveFormat" -as [type]
+$word = New-Object -ComObject word.application
+$word.visible = $false
+
+# Make sure "Store random number..." setting is off 
+$word.Options.StoreRSIDOnSave = $false
```

Comment:
This is the setting that we need to change before saving the file. It's application-wide so should stick once set, but can't hurt to set it explicitly each time. Can see comments on @macmillanpublishers/bookmaker#128 for path to actual option in Word interface.